cmd/yoke/cmd_takeoff.go

```diff
@@ -130,6 +135,27 @@ func TakeOff(ctx context.Context, params TakeoffParams) error {
 		return ExportToFS(params.Out, params.Release, resources)
 	}
 
+	if params.DiffOnly {
+		revisions, err := kube.GetRevisions(ctx, params.Release)
+		if err != nil {
+			return fmt.Errorf("failed to get revision history: %w", err)
+		}
+		latestAppliedRevision := revisions.History[len(revisions.History)-1]
+
+		a, err := text.ToYamlFile(fmt.Sprintf("revision %d", latestAppliedRevision.ID), latestAppliedRevision.Resources)
```

Comment:
instead of revision numbers as text the titles, I would use something more semantic like `current` and `next`.